perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -88,6 +100,36 @@
 
 	private final static Set<String> extSourcesWithMultipleIdentifiers = BeansUtils.getCoreConfig().getExtSourcesMultipleIdentifiers();
 
+	/**
+	 * Create a list from the given objects and then check if the principal is authorized.
```

Comment:
Same problem like in Entry layer. This javadoc is confusing. Why is important of creating a list? How does one should call this method to get correct results?

Reply:
Method authorized() now takes only List<PerunBean>, so this logic was erased.